.drone.yml

```diff
@@ -1,6 +1,13 @@
 workspace:
+  base: /srv/app
   path: src
 
+clone:
+  git:
+    image: plugins/git:latest
```

Comment:
we can ignore `:latest` since the default behavior is to download the latest tag.